fairlearn/reductions/exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -62,9 +63,10 @@ def __init__(self, estimator, constraints, eps=0.01, T=50, nu=None, eta_mul=2.0)
         self._best_classifier = None
         self._classifiers = None
 
-    def fit(self, X, y, group_data=None, **kwargs):
+    def fit(self, X, y, **kwargs):
         # TODO: validate input data; unify between grid search and expgrad?
-        self._expgrad_result = exponentiated_gradient(X, group_data, y, self._estimator,
+        self._expgrad_result = exponentiated_gradient(X, kwargs[_KW_GROUP_MEMBERSHIP], y,
```

Comment:
oh this is rough. There's no documentation of the argument, but then we'll fail right here when using it because kwargs won't have an entry named like that.

Reply:
[the truth is even more confusing... _KW_GROUP_MEMBERSHIP="sensitive_features"; so the constant is being renamed to _KW_SENSITIVE_FEATURES]